maestro-ios/src/main/java/ios/simctl/SimctlIOSDevice.kt

```diff
@@ -86,7 +86,10 @@ class SimctlIOSDevice(
     }
 
     override fun openLink(link: String): Result<Unit, Throwable> {
-        TODO("Not yet implemented")
+        runCatching {
+            Simctl.openURL(deviceId, link)
+        }
+        return Ok(Unit)
```

Comment:
Same, this should be:

Returning result of runCatching:

runCatching {
    Simctl.openURL(deviceId, link)
}
The result of above is either Ok or Err that helps caller telling if the call actually succeeded or failed

Reply:
Aha, I'll update all functions in SimctlIOSDevice that are applicable in a separate PR